src/trix/models/composition.coffee

```diff
@@ -76,25 +76,40 @@ class Trix.Composition extends Trix.BasicObject
     document = @document
     {index, offset} = document.locationFromPosition(position)
     block = document.getBlockAtIndex(index)
+    previousCharacter = block.getPreviousCharacter(offset)
+    nextCharacter = block.getNextCharacter(offset)
```

Comment:
Since the variables here are clearly named, I'd remove the new `Block` methods and use `Text#getStringAtRange` directly:
```coffee
[previousCharacter, nextCharacter] = block.text.getStringAtRange([offset - 1, offset + 1]).split("")
```